services/brig/src/Brig/User/API/Auth.hs

```diff
@@ -145,6 +145,28 @@ routes = do
 
     -- Internal
 
+    post "/i/users/backdoor-login" (continue backdoorLogin) $
```

Comment:
Why not `/i/login`, in line with `/login`?  `/i/backdoor-login` would be ok with me, too.  @tiago-loureiro?

Reply:
Fixed.